src/types/routeProps.ts

```diff
@@ -7,9 +7,7 @@ import { Query } from '@/utilities/query'
 
 export type RouteComponent = Component | DefineComponent | AsyncComponentLoader
 
-export interface RouteMeta {
-
-}
+export interface RouteMeta extends Record<string, unknown> {}
```

Comment:
Interesting. I'm surprised this works

Reply:
I did test it lol

Reply:
Yeah I assumed it worked. Just wouldn't have thought that